### DISCUSSION TITLE
The command texts aren't show

### DISCUSSION THREAD

**Discussion by tisaii** [ohmyzsh/ohmyzsh#11744] [Help] [answered]
there aren't any commands font shown when using zsh. The commands are invisible but the results correctly show.
I used to try changing the Linux's encoding and re-installing the zsh or oh-my-zsh. but it doesn't work.
Other machines are correct to work when using zsh but this one isn't.        Thanks for your help.
For example, it's showing like this:
                    (I input command "ll" but isn't shown)
total 420K          (results show)                                                                                                                                          
drwxr-xr-x  6 root root   95 Jun 10 12:28 docker

and like this:
                  (I input multiple enter but only shows one blank line)

**Comment by mcornella** [comment]
Do you have `stty -echo` somewhere in your configuration? That turns off showing what you write.

**Reply by tisaii** [reply, marked as answer]
uh...I find zsh can work when I use the other shell, It looks like caused by FinalShell, but it's so weird that the other machines can normally work in FinalShell with the same config(Encoding, etc.).
anyway, thanks for your reply.
